Lib/test/test_shutil.py

```diff
@@ -1604,6 +1607,16 @@ def test_empty_path_no_PATH(self):
             self.assertIsNone(rv)
 
 
+class TestWhichBytes(TestWhich):
+    def setUp(self):
+        TestWhich.setUp(self)
+        self.dir = os.fsencode(self.dir)
+        self.file = os.fsencode(self.file)
+        self.temp_file.name = os.fsencode(self.temp_file.name)
+        self.curdir = os.fsencode(self.curdir)
+        self.ext = os.fsencode(".EXE")
```

Comment:
nitpick!

```suggestion
        self.ext = os.fsencode(self.ext)
```

Reply:
Sorry about that.  I meant to do that on the last push.